README.md

```diff
@@ -138,7 +138,7 @@ These are the base data types for defining circuit components:
 
 ![Image](https://raw.githubusercontent.com/freechipsproject/chisel3/master/doc/images/type_hierarchy.svg?sanitize=true)
 
-## Developer Documentation
+## Chisel Library Developer Documentation
```

Comment:
Just curious: what does "Chisel Library" refer to here?

Reply:
What is Chisel? It is a Scala library, right? But I am open to a better name. Just Chisel Developer Documentation is not enough. If I am a Java developer I develop Java programs and not the Java compiler or the JVM. If I am a Chisel developer than I am developing hardware in Chisel, not the Chisel language/library. So maybe: "Chisel Language Developer Documentation"?

Reply:
From a software perspective I thought "developer doc" was fine since it connoted the opposite of "user doc".

I almost thought "Chisel Library Developer Documentation" was "how to develop a library e.g. rocketchip that uses Chisel"

Reply:
I get the same impression as @edwardcwang, Chisel is technically a library but I find most new users don't really grasp that for a while. I've noticed that a lot of projects use the word "Contributor", so what about: "Contributor Documentation"?

The next line mentions "This section describes how to get started developing Chisel itself", but maybe we should slightly rephrase that and bold it.